caps/misc_test.go

```diff
@@ -42,8 +40,9 @@ func (s *MiscSuite) TestLoadBuiltInTypes(c *C) {
 	repo := NewRepository()
 	err := LoadBuiltInTypes(repo)
 	c.Assert(err, IsNil)
-	c.Assert(repo.types, testutil.Contains, BoolFileType)
-	c.Assert(repo.types, HasLen, 1) // Update this whenever new built-in type is added
+	c.Assert(repo.types, DeepEquals, []Type{
```

Comment:
For the record, I generally try to avoid such whitebox tests. There's really not much benefit in simply knowing something is inside a given attribute. The best kind of test is those that verify that the value interface behaves as it should. In other words, what is the observable result of this repository knowing about the bool-file type? This outcome cannot change because it's a promise we're making to clients of this API, so it's a better thing to ensure in a test too. The internals can be modified as long as this promise isn't broken.